apps/server/src/modules/tool/uc/mapper/contextType.mapper.ts

```diff
@@ -0,0 +1,12 @@
+import { ToolContextType } from '@src/modules/tool/interface';
```

Comment:
Please use ../.. when the files are in the module self

Reply:
done